client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -215,8 +223,13 @@ export default memo( function SitesTableRow( { site }: SiteTableRowProps ) {
 							<TransferNoticeWrapper { ...result } />
 						) : (
 							<>
-								{ translatedStatus }
-								<SiteLaunchNag site={ site } />
+								<div>
+									{ translatedStatus }
+									<SiteLaunchNag site={ site } />
+								</div>
+								{ isDIFMInProgress && (
+									<BadgeDIFM className="site__badge">{ __( 'Built By Express' ) }</BadgeDIFM>
```

Comment:
Should the copy be updated in-line with #88540? 

```suggestion
									<BadgeDIFM className="site__badge">{ __( 'Express Website Design Service' ) }</BadgeDIFM>
```

Reply:
Hi! That's a good spot! The change in https://github.com/Automattic/wp-calypso/pull/88540 is still under discussion, and I'll update this string later if this PR gets merged before #88540. Thanks!